Sources/NIO/System.swift

```diff
@@ -334,7 +335,7 @@ internal enum Posix {
     }
 
     @inline(never)
-    public static func open(file: UnsafePointer<CChar>, oFlag: CInt, mode: mode_t) throws -> CInt {
+    public static func open(file: UnsafePointer<CChar>, oFlag: CInt, mode: CInt) throws -> CInt {
```

Comment:
This change is wrong: `mode_t` exists on the Posix platforms and is defined as `UInt32`, at least on Linux. It'd be better for this file to define `mode_t` for Windows as the right thing.

Reply:
Sure, but that wasn't meant to be part of this change, I want to try to do the `mode_t` changes together as this isn't the only site.